packages/atomic/src/pages/examples/insights.html

```diff
@@ -441,6 +457,18 @@
       document.addEventListener('atomicInsightResultActionClicked', (event) => {
         console.log('Result action clicked', event.detail);
       });
+
+      document.addEventListener('atomic/insight/attachToCase/attach', (event) => {
+        console.log('attach', event.detail);
+        const {controller, result} = event.detail;
+        controller.attach();
+      });
+
+      document.addEventListener('atomic/insight/attachToCase/detach', (event) => {
+        console.log('detach', event.detail);
+        const {controller, result} = event.detail;
+        controller.detach();
+      });
```

Comment:
Can you explain why we need those javascript event listeners ? Couldn't this be done via stencil events and without those events in the implementation file ? 

https://stenciljs.com/docs/events